pkg/logcli/client/client.go

```diff
@@ -119,7 +130,22 @@ func (c *Client) Series(matchers []string, from, through time.Time, quiet bool)
 	return &seriesResponse, nil
 }
 
-func (c *Client) doQuery(path string, query string, quiet bool) (*loghttp.QueryResponse, error) {
+// LiveTailQueryConn uses /api/prom/tail to set up a websocket connection and returns it
+func (c *DefaultClient) LiveTailQueryConn(queryStr string, delayFor int, limit int, from int64, quiet bool) (*websocket.Conn, error) {
```

Comment:
I only moved this to keep it in the same order as the interface and put it before private methods